tests/components/pvpc_hourly_pricing/test_sensor.py

```diff
@@ -59,27 +61,27 @@ def mock_now():
         )
         assert num_warnings == 1
         assert num_errors == 0
-        assert pvpc_aioclient_mock.call_count == 7
+        assert pvpc_aioclient_mock.call_count == 9
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5232061)